### DISCUSSION TITLE
Feature request: Backup media

### DISCUSSION THREAD

**Discussion by maikschneider** [sourcebroker/deployer-extended-media#7] [Ideas]
Hi, 

Great work again! 
I really would love to have a backup task, like the one in the extended database project. 
I looked into it and startet creating the first `media:export` task [right here](https://github.com/maikschneider/deployer-extended-media/commit/e858d69e703112cc7cc9d6c92f0e4adffd448dd8).
Analog to the database tasks, I would continue with a `media:compress`, `media:dumpclean` and `media:backup`.

Maybe you are interested to have look over it and merge it.

Later, I would like to have a configurable task for syncing the backups to an external backup storage.

Cheers!

**Comment by kszymukowicz** [comment]
hi

in deployer-extended there is already file backup task. 
https://github.com/sourcebroker/deployer-extended#filebackup

It was more implemented to store some smaller files that are outside git. 
For example could be ext:form files in fileadmin or language files. 

However you can probably use it also for media.

btw: for media it would be good to have incremental backup.
